storage/allocator_test.go

```diff
@@ -777,7 +683,10 @@ func Example_rebalancing() {
 	// Model a set of stores in a cluster,
 	// randomly adding / removing stores and adding bytes.
 	g := gossip.New(nil, 0, nil)
-	alloc := newAllocator(g)
+	stopper := stop.NewStopper()
+	defer stopper.Stop()
+	sp := NewStorePool(g, DefaultTimeUntilStoreDead, stopper)
```

Comment:
The test one may cause some of the stores to be marked as dead.  We're not testing that here.